src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -409,8 +417,117 @@ public Object call() throws WorkflowException {
             throw new RuntimeException("cancelDVAsSystemUser() Unable to cancel DV: " + dvDocumentNumber, e);
         }
     }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public boolean autoApproveDisbursementVouchersSpawnedByRecurringDvs() {
```

Comment:
This function is a bit long, can it be refactored a bit more to break things up?